env0/templates.go

```diff
@@ -45,17 +45,21 @@ func getTemplateSchema(templateType TemplateType) map[string]*schema.Schema {
 		"is_github_enterprise",
 	}
 
+	appendByType := func(strs []string, str string) []string {
+		if templateType == TemplateTypeShared {
+			return append(strs, str)
+		} else {
+			return append(strs, "template.0."+str)
+		}
+	}
+
 	allVCSAttributesBut := func(strs ...string) []string {
 		sort.Strings(strs)
 		ret := []string{}
 
 		for _, attr := range allVCSAttributes {
 			if sort.SearchStrings(strs, attr) >= len(strs) {
-				if templateType == TemplateTypeShared {
-					ret = append(ret, attr)
-				} else {
-					ret = append(ret, "template.0."+attr)
-				}
+				ret = appendByType(ret, attr)
```

Comment:
I'm not sure what is `ret`. can you rename and use full words when naming variables? 🙏🏼

Reply:
sure!